docs/topics/Threads.md

```diff
@@ -133,54 +119,114 @@ There are four new `GET` routes for enumerating threads in a specific channel:
 
 Webhooks can send messages to threads by using the `thread_id` query parameter. See the [execute webhook](#DOCS_RESOURCES_WEBHOOK/execute-webhook) docs for more details.
 
-## Additional context on the the THREAD_LIST_SYNC and THREAD_CREATE dispatches
+## Forums
+
+A `GUILD_FORUM` (type `15`) channel is similar to a `GUILD_TEXT` channel, except *only* threads can be created in that channel. Unless otherwise noted, threads in forum channels behave in the same way as in text channels—meaning they use the same endpoints and receive the same Gateway events.
```

Comment:
maybe mention tags here
```suggestion
A `GUILD_FORUM` (type `15`) channel is similar to a `GUILD_TEXT` channel, except *only* threads can be created in that channel, and forum channels support organizing their threads with tags. Unless otherwise noted, threads in forum channels behave in the same way as in text channels—meaning they use the same endpoints and receive the same Gateway events.
```